lib/location.js

```diff
@@ -261,9 +275,16 @@ var LocationState = State.extend(function (self, name, opts) {
         return '\n' + number + '. ' + address.label;
     };
 
-    self.generate_footer = function() {
-        return '\n' + 'n. '+self.next_text + '\n' + 'p. '+self.previous_text;
-
+    self.generate_footer = function(content) {
+        var options = [];
+        if (self.metadata.current_page >= self.metadata.pages.length && self.retry_text){
```

Comment:
What you describe is sensible, but that's not what the code currently does. 

I would vote for the `r. Retry` to be on every page if it's active (otherwise one can be stuck on page 3/5 wanting to retry?). Thoughts?